tests/test_views.py

```diff
@@ -1,15 +1,32 @@
 import pytest
-
+from django.urls import reverse
+from collections import OrderedDict
 
 def test_api_parse_succeds(client):
-    # TODO: Finish this test. Send a request to the API and confirm that the
-    # data comes back in the appropriate format.
     address_string = '123 main st chicago il'
-    pytest.fail()
+
+    expected_return_input_string = '123 main st chicago il'
+    expected_return_address_components = OrderedDict([('AddressNumber', '123'), ('StreetName', 'main'), ('StreetNamePostType', 'st'), ('PlaceName', 'chicago'), ('StateName', 'il')])
+    excpected_return_address_type = 'Street Address'
```

Comment:
Looks like you've got a typo in the variable name here.

Reply:
Good catch, I need to install a spell checker into my IDE! (Updated [here](https://github.com/noah-marcus/code-challenge/commit/230e1d11c2b452594cbc73b0aaf7a231a5e2882d))